shiftcommerce/shift-react-components
# Issue: Checkout Enhancements

## Overview

Github Issue: https://github.com/shiftcommerce/shift-front-end-react/issues/865

This PR adds the `PaymentMethodSummary` component and styling updates required for the PayPal integration feature. It also renames `Checkout 4 - Payment` components from 

```
PaymentMethod => Payment
PaymentMethodHeader => PaymentHeader
PaymentMethodSummary => PaymentSummary
```

The old names are now being used for `Checkout 1 - Payment Method` components
 
It also bumps the package version to `v0.13.0`

## QA

1. checkout branch `issue/865_checkout_enhancements`

2. Add items to cart and navigate to the first step of checkout - `payment-method` selection

3. Select `Pay By Credit/Debit Card` option and ensure the payment methods summary component is displayed on `Step 2 - Shipping Address`

![Payment Methods Summary for Card](https://user-images.githubusercontent.com/4486874/56129680-adeab600-5f7a-11e9-8744-7d668d78d143.png)

4. Navigate to `Step 1 - Payment Methods` and select `PayPal` option. When you are redirected back, ensure that the payment methods summary component is displayed on `Step 2 - Shipping Address`

![Payment Methods Summary for PayPal](https://user-images.githubusercontent.com/4486874/56129788-002bd700-5f7b-11e9-8e90-e07ad5acba57.png)

5. Also check that there is a gap in between the shopping basket buttons - 

![Gap Between Buttons](https://user-images.githubusercontent.com/4486874/56129871-48e39000-5f7b-11e9-8b82-e4c5d119d90c.png)

6. Check the Payment Method Summary component on the Review page for both checkout options

![Card](https://user-images.githubusercontent.com/4486874/56138252-6f5ef680-5f8e-11e9-8c63-af816c3e1b2f.png)

![PayPal](https://user-images.githubusercontent.com/4486874/56138253-6ff78d00-5f8e-11e9-80e2-c383f1ad52e9.png)
